linter.py

```diff
@@ -46,56 +40,24 @@ class ESLint(NodeLinter):
     }
 
     def find_errors(self, output):
-        """
-        Parse errors from linter's output.
+        """Parse errors from linter's output.
 
-        We override this method to handle parsing eslint crashes.
+        Log errors when eslint crashes or can't find its configuration.
         """
-
-        match = self.config_fail_regex.match(output)
-        if match:
-            return [(match, 0, None, "", "config", match.group(1), None)]
         match = self.crash_regex.match(output)
         if match:
-            return [(match, 0, None, "exception", "", match.group(2), None)]
+            logger.error(output)
+            return []
 
         return super().find_errors(output)
 
     def split_match(self, match):
```

Comment:
Do we really need this?

Reply:
If there is an error we need to log it. What else should we do? Before, it marked the first line of the view.

Reply:
The log is only in debug mode right? We already print all output then, no need to capture and match it.

Reply:
Or is logger.error always printed? In that case 👍🏻👍🏻